codegen_glibmm/templates/stub.h.templ

```diff
@@ -86,10 +86,15 @@ protected:
 private:
     bool emitSignal(const std::string &propName, Glib::VariantBase &value);
 
+    class RegisteredObject {
+    public:
+        guint id;
```

Comment:
`guint id = 0;`? (Not really relevant now but better with defined than undefined defaults is code ever changes. :)

Reply:
I had to revert this, because it prevents initializing the object as:

    m_registered_objects.emplace_back(RegisteredObject {
        registration_id,
        connection,
        object_path
    });